Navidux/Sources/Navidux/Coordinator.swift

```diff
@@ -0,0 +1,17 @@
+import UIKit
+
+/// Main Object that used in navigation. Have many different types of presentation and inner methods to manipulate navigation stack.
+/// Work on states with associated values.
+/// - Note use **actionReducer(action:)** function with some action to change state of navigation stack. It's trigger inner function to change navigation screen.
+/// - Note use **start()** function to trigger first default screen or **actionReducer(action:)** with **restruct** action to set up inital navigation stack state.
+public protocol Coordinator: AnyObject {    
+    func actionReducer(action: Navigation.Action)
```

Comment:
I think there i some issues with codestyle. Mainly there should be new line after protocol declaration and first method/property.

Reply:
RMR-code-style don't have any appointments about protocols. In another 'Swift Style Guide' (https://google.github.io/swift/ from google) I found some examples with protocols. It's normal not to have new line after protocol declaration.

Reply:
Oh, I'm ok with it then